akka-actor/src/main/java/akka/actor/AbstractScheduler.java

```diff
@@ -33,15 +34,44 @@
    */
   @Override
   public abstract Cancellable schedule(FiniteDuration initialDelay,
-      FiniteDuration interval, Runnable runnable, ExecutionContext executor);
+                                       FiniteDuration interval,
+                                       Runnable runnable,
+                                       ExecutionContext executor);
+
+  /**
+   * Schedules a function to be run repeatedly with an initial delay and
+   * a frequency. E.g. if you would like the function to be run after 2
+   * seconds and thereafter every 100ms you would set delay = Duration(2,
+   * TimeUnit.SECONDS) and interval = Duration.ofMillis(100)
+   */
+  public Cancellable schedule(final java.time.Duration initialDelay,
```

Comment:
@hepin1989  This will not be visible as Java API because the exposed API is `Scheduler`, e.g. `system.scheduler`.

The only purpose of AbstractScheduler is if someone would like to implement their own scheduler in Java.

Reply:
Okay, I am following that.